cogs/christmas_2021.py

```diff
@@ -102,87 +102,105 @@ async def addbox(self, ctx, user: FetchUserConverter, box_type, num: int = 1):
 
     @checks.has_started()
     @commands.max_concurrency(1, commands.BucketType.user)
-    @christmas.command()
-    async def open(self, ctx, box_type):
+    @christmas.command(aliases=("o"))
+    async def open(self, ctx, box_type: str = "", amt: int = 1):
         """Open a box"""
 
-        member = await self.bot.mongo.fetch_member_info(ctx.author)
+        if amt <= 0:
+            return await ctx.send("Nice try...")
 
+        if amt > 15:
+            return await ctx.send("You can only open 15 event boxes at once!")
```

Comment:
Maybe `"You can only open up to 15 event boxes at once!"`

Reply:
gotcha!